tests/unit/main.fmf

```diff
@@ -32,9 +32,8 @@ require+:
     enabled: false
 
     adjust+:
-      - when: initiator is not defined or distro == fedora-39
-        because: Enable locally or in CI on Fedora 39
-
+      - when: initiator is not defined
```

Comment:
`distro` should be preserved, with a newer one. The goal is to run it on just once, as there's no need to run this test on all distros we run tests on, and `fedora-39` was the chosen one. Today, I suppose `fedora-41` should step in and take over.

Reply:
Ah, I see. What about using `fedora-rawhide` then, so that we don't have to keep updating the value?

Reply:
Maybe. I believe I was aiming at some stable distro, but it should not matter as it uses PyPI. System packages might cause trouble just like anything else in rawhide, but that's all.

Reply:
It could look like this: dd58f215